openwisp_radius/api/views.py

```diff
@@ -159,6 +160,23 @@ def get(self, request, *args, **kwargs):
 download_rad_batch_pdf = DownloadRadiusBatchPdfView.as_view()
 
 
+class UserLanguageMixin(object):
+    def save_language(self, user):
+        languages = self._parse_language_header(self.request)
+        settings_languages = list(zip(*settings.LANGUAGES))[0]
+        for language in languages:
+            if language == user.language:
+                break
+            elif language in settings_languages:
+                user.language = language
+                user.save()
+                break
```

Comment:
I'd like to avoid looping on all the languages in settings.LANGUAGES since there's many entries there, it looks like django's `get_language_from_request` already does this work in the most possible efficient way and we should find a way to leverage that instead of reimplementing it.